src/models/send-to-api.js

```diff
@@ -23,12 +23,22 @@ async function sendToApi (requestParams) {
     .then(prefixUrl(conf.API_HOST))
     .then(addOauthHeader(tokens))
     .then((requestParams) => {
-      if (process.env.CLEVER_VERBOSE) {
-        Logger.debug(`${requestParams.method.toUpperCase()} ${requestParams.url} ? ${JSON.stringify(requestParams.queryParams)}`);
-      }
+      Logger.debug(`${requestParams.method.toUpperCase()} ${requestParams.url} ? ${JSON.stringify(requestParams.queryParams)}`);
       return requestParams;
     })
-    .then((requestParams) => request(requestParams, { retry: 1 }));
+    .then(request)
+    .catch(processError);
+}
+
+function processError (error) {
```

Comment:
In QA, I never get those kind of error logs: 

```node
[DEBUG] Logs stream (error) no healthcheck since 4390, restarting...
[DEBUG] Logs stream (error) fetch failed
[DEBUG] Logs stream (error) fetch failed
[DEBUG] Logs stream (error) fetch failed
[DEBUG] Logs stream (error) fetch failed
[DEBUG] Logs stream (error) fetch failed
[STACKTRACE]
Error: max retry count reached: TypeError: fetch failed
[/STACKTRACE]
[ERROR] Error: max retry count reached: TypeError: fetch failed
```